plugin/pkg/admission/limitranger/admission.go

```diff
@@ -162,7 +163,10 @@ func defaultContainerResourceRequirements(limitRange *api.LimitRange) api.Resour
 }
 
 // mergePodResourceRequirements merges enumerated requirements with default requirements
-func mergePodResourceRequirements(pod *api.Pod, defaultRequirements *api.ResourceRequirements) {
+// it returns a slice for what request and what limit values were merged
+func mergePodResourceRequirements(pod *api.Pod, defaultRequirements *api.ResourceRequirements) (requests []string, limits []string) {
+	requests = []string{}
```

Comment:
these two aren't necessary afaik, they'll be nil by default, and append will work.